app/controllers/admin/orders_controller.rb

```diff
@@ -1,9 +1,23 @@
 # frozen_string_literal: true
 
+require 'csv'
+
 module Admin
   class OrdersController < AdminController
     def index
       @orders = Order.includes(:employee).order(:delivered)
     end
+
+    def export
+      @orders = Order.order(:delivered)
```

Comment:
don't you have n+1 here?
```suggestion
      @orders = Order.includes(:employee).order(:delivered)
```

Reply:
yes, n+1 here :)
corrected